css/style.css

```diff
@@ -1039,22 +1041,1370 @@ li { list-style-type: none; }
+    line-height: 18px;
+    font-style: normal;
+    font-weight: 600;
+    text-align: center;
+  }
+  
+  .time-speed__text {
+    display: flex;
+    align-items: flex-start;
+    width: 28px;
+    height: 19px;
+    margin: 19px 0 0 0;
+    padding: 0;
+  }
+  
+  .subtext {
+    font-size: 10px;
+    line-height: 12px;
+    font-style: normal;
+    font-weight: normal;
+  }
+  
+  .time-speed__subtext {
+    display: flex;
+    align-content: flex-start;
+    width: 64px;
+    height: 13px;
+    margin: -13px 0 0 52px;
+  }
+
+  .distance {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: flex-start;
+    align-content: flex-start;
+    width: 115px;
+    height: 53px;
+    margin: 0 14px 0 0;
+    padding: 0;
+  }
+
+  .distance__line {
+    display: flex;
+    width: 25.29px;
+    height: 41px;
+    margin: 10px 11.71px 2px 0;
+    background: url(https://i.ibb.co/JK7BzBg/Group-2-mob.png) no-repeat;
+    background-size: 100% 90%;
+    background-position: center bottom;
+  }
+  
+  .distance__numeral {
+    display: flex;
+    width: 49px;
+    height: 43px;
+    margin: 0 7px 0 0;
+    padding: 2px 0 0 0;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 27px;
+    line-height: 33px;
+    align-items: center;
+  }
+  
+  .distance__text {
+    display: flex;
+    align-content: flex-start;
+    width: 22px;
+    height: 19px;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 18px;
+    margin: 19px 0 0;
+    padding: 0;
+  }
+  
+  .distance__subtext {
+    display: flex;
+    width: 56px;
+    height: 13px;
+    font-style: normal;
+    font-weight: normal;
+    text-align: left;
+    font-size: 10px;
+    line-height: 12px;
+    margin: -12px 0 0 37px;
+    padding: 0;
+  }
+
+  .button {
+    display: flex;
+    width: 160px;
+    height: 40px;
+    font-family: "Montserrat", sans-serif;
+    text-align: center;
+    font-weight: bold;
+    text-transform: uppercase;
+  }
+  
+  .test-drive {
+    display: flex;
+    flex-flow: row wrap;
+    width: 303px;
+    height: 40px;
+    justify-content: center;
+    align-content: center;
+    margin: 0;
+  }
+
+  .test-drive__rudder {
+    display: flex;
+    width: 22px;
+    height: 22px;
+    margin: 0 7px 0 0;
+    padding: 0;
+    background: url(https://i.ibb.co/Stm3197/2-7-red-wheel.png) no-repeat;
+    background-size: 100%;
+    background-position: center;
+  }
+
+  .test-drive__link {
+    display: flex;
+    width: 89px;
+    height: 18px;
+    font-size: 13px;
+    line-height: 16px;
+    font-style: normal;
+    font-weight: bold;
+    text-decoration: none;
+    color: #e61234;
+    margin: 0 25px 0 0;
+    text-transform: uppercase;
+  }
+
+  .test-drive__link:hover {
+    display: flex;
+    font-size: 14px;
+    line-height: 16px;
+    font-weight: bold;
+    text-align: center;
+    text-decoration: none;
+    color: #000;
+    cursor: pointer;
+    padding: 0;
+    border: 0;
+  }  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Hi-Tesla/pullRequest?prid=5851407)